src/main/java/the/bytecode/club/bytecodeviewer/BytecodeViewer.java

```diff
@@ -196,7 +196,7 @@ public static void main(String[] args)
             
             //setup swing components
             viewer = new MainViewerGUI();
-            SwingUtilities.updateComponentTreeUI(viewer);
+            //SwingUtilities.updateComponentTreeUI(viewer);
```

Comment:
I don't know if this is important. On my machine, this line results in the following error:
```
java.lang.IllegalArgumentException: listener mismatch
        at java.desktop/java.awt.dnd.DropTarget.removeDropTargetListener(DropTarget.java:334)
        at com.github.weisj.darklaf.ui.tabbedpane.DarkTabbedPaneUI.uninstallUI(DarkTabbedPaneUI.java:155)
        at java.desktop/javax.swing.JComponent.uninstallUIAndProperties(JComponent.java:745)
        at java.desktop/javax.swing.JComponent.setUI(JComponent.java:720)
        at java.desktop/javax.swing.JTabbedPane.setUI(JTabbedPane.java:249)
        at java.desktop/javax.swing.JTabbedPane.updateUI(JTabbedPane.java:265)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1363)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI0(SwingUtilities.java:1378)
        at java.desktop/javax.swing.SwingUtilities.updateComponentTreeUI(SwingUtilities.java:1354)
        at the.bytecode.club.bytecodeviewer.BytecodeViewer.main(BytecodeViewer.java:199)
```